docs/manual-CN/1.overview/1.concepts/1.data-model.md

```diff
@@ -22,17 +22,17 @@ G = < V, E, P<sub>V</sub>, P<sub>E</sub> >,
 
 节点用于表示现实世界中的实体，本例的数据中共包含 11 个节点。
 
-<img src="https://user-images.githubusercontent.com/42762957/64932628-00eecf00-d873-11e9-844b-6b2a535ca734.png" width="10%" height="20%">
+<img src="https://user-images.githubusercontent.com/42762957/64932628-00eecf00-d873-11e9-844b-6b2a535ca734.png" width="15%" height="20%">
 
 ## 标签
 
  **Nebula Graph** 使用**标签**对节点进行分类，本例包含的节点标签为 **player** 和 **team**。
-<img src="https://user-images.githubusercontent.com/42762957/64932330-bff5bb00-d870-11e9-9940-4ff76ceca353.png" width="35%" height="20%">
+<img src="https://user-images.githubusercontent.com/42762957/64932330-bff5bb00-d870-11e9-9940-4ff76ceca353.png" width="50%" height="25%">
 
 ## 边
 
 边用来连接节点，边通常表示两个节点间的某种关系或行为，本例中的边为 _**serve**_ 和 _**like**_。
```

Comment:
"like" is not used as discussed.

Reply:
Good catch thanks, will fix in another pr.